### PR TITLE
Fix two failing tests and add a custom test reporter

### PR DESCRIPTION
**Pull request by mokimo**
- Fix two failing tests
- Use a custom test reporter along with the defaultReporter that can miss to log some tests. 

Ticket: https://jira.corp.adobe.com/browse/MWPW-131198
Note: A PR to fix this bug in the defaultReporter has been opened as well https://github.com/modernweb-dev/web/pull/2278

### CODE REVIEW

**Review by overmyheadandbody — APPROVED** [review]

File: web-test-runner.config.mjs [hunk 1]
```diff
@@ -1,5 +1,19 @@
 import { importMapsPlugin } from '@web/dev-server-import-maps';
+import { defaultReporter } from '@web/test-runner';
 
+function customReporter() {
+  return {
+    async reportTestFileResults({ logger, sessionsForTestFile }) {
+      sessionsForTestFile.forEach((session) => {
+        session.testResults.tests.forEach((test) => {
+          if (!test.passed && !test.skipped) {
+            logger.log(test);
```
Comment (narcis-radu) [inline web-test-runner.config.mjs:10]: 👍

**Review by narcis-radu — APPROVED** [review]

**Review by honstar — APPROVED** [review]

**Review by auniverseaway — CHANGES_REQUESTED** [review]
We want to fix the root problem and not introduce a custom reporter that we have to maintain.

**Comment by mokimo** [comment]
@auniverseaway The root problem is a bug/oversight in the custom reporter, for which I've opened a PR.

https://github.com/modernweb-dev/web/pull/2278 - since we can't control when/if that is getting merged, there's a lot of value in having that customReporter in the meantime IMO.

**Comment by auniverseaway** [comment]
@mokimo My concern was that we were introducing a _replacement_ reporter, not an _additional_ reporter.

We don't want to be on the hook for maintaining a custom reporter over a couple of incorrectly written tests. Re-looking at what you have, I think we're fine... assuming my comment above is accurate.

Thanks for looking into this and sending a PR. Really appreciate it.

**Review by auniverseaway — APPROVED** [review]

**Pull request merged**